packages/api/src/external/commonwell/proxy/cw-process-request.ts

```diff
@@ -38,30 +41,41 @@ const allowedQueryParams = ["status", "date", "_include", "_summary"];
  * The main function, it will:
  * - get the Patient ID and the Customer ID (based on the Org OID) from the request;
  * - remove invalid parameters;
- * - query the FHIR server (paginated);
- * - return a bundle with the consolidated FHIR Resources.
+ * - query s3 for metadata files;
+ * - parse them and create FHIR DocumentReferences for them
+ * - return a bundle with the DocumentReferences
  */
 export async function processRequest(req: Request): Promise<Bundle<Resource>> {
   log(`ORIGINAL URL: ${req.url}`);
 
   const { cxId, patientId } = await getPatientAndCxFromRequest(req);
-
   const { resource, count, params } = fromHttpRequestToFHIR(req);
 
+  // TODO: We might need to introduce filtering based on the provided query params.
   log(
     `UPDATED resource: ${resource} / cx ${cxId} / patient ${patientId} ` +
       `/ count : ${count}, params: ${params.toString()}`
   );
```

Comment:
These params are now only used in this log, so we'll always be returning everything we have that our CX uploaded. Idk how important this is right now to introduce filtering based on these params to the new flow. The only one I'm seeing that makes sense to me is the `date` filter.

Reply:
This is a good point. The previous version was applying most filters to the call to the FHIR server.

Why can't we also apply those filters here?